.github/workflows/verify.yaml

```diff
@@ -36,7 +36,10 @@ jobs:
       - name: Run Tests (Vert.x 3 Mem)
         run: mvn test install -Dinvoker.skip -Dmaven.javadoc.skip=true -B -Dapiman.gateway-test.config=vertx3-mem
       - name: Run Tests (Vert.x 3 ES)
-        run: mvn test install -Dinvoker.skip -Dmaven.javadoc.skip=true -B -Dapiman.gateway-test.config=vertx3-es
+        run: |
+          docker run -d -p 19250:9200 -p 9300:9300 -e "discovery.type=single-node" --name=elasticsearch-7 docker.elastic.co/elasticsearch/elasticsearch-oss:7.6.2
```

Comment:
Why is this not needed for the `servlet-es` test variant?

Reply:
I will take a further look tomorrow, the tests for `vert.x` and `servlet-es` work not in a consistent way. Historically I think the embedded elasticsearch library was only used in the `servlet-es` variant. But I will check tomorrow. Maybe we can improve the test framework further, this was not part of our elasticsearch upgrade so far.

Reply:
It wasn't really a request for a change, more a matter of curiosity.  :)